src/App/CabanasRD/Messages/DisplayAlertCommonMessages.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+namespace CabanasRD
+{
+    public static class CommonDisplayAlertMessages
```

Comment:
Como solución temporal lo veo bien, luego abrimos un issue para mover los strings a un resx file.

Reply:
Sounds pretty cool

I didn't know we can add resx file to hold common strings and share between Android and iOS projets.